massgov/mayflower-patternlab
# add additional link for linked page banner

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Adds support for two linked entities in the linked page-banner variation. It's achieved by moving from simply a `link` attribute to `primaryLink` and `secondaryLink` attributes which each have their own descriptions. Omitting the HREF attribute in either link makes the 'link' plain text. 


## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8711)


## Steps to Test


1. View `http://localhost:3000/?p=pages-organization-elected-official`
2. Ensure functionality of the 'links' in the page banner is as expected.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Currently, this change doesn't support the old `link` attribute data model, `link` would have to changed to `primaryLink` and the `description` moved to be `primaryLink.description`.


#### Impacted Areas in Application


* `@organisms/page-banner`
